content/reference-rippled.md

```diff
@@ -8488,36 +8488,40 @@ The `info` object may have some arrangement of the following fields:
+| `complete_ledgers`                  | String                    | Range expression indicating the sequence numbers of the ledger versions the local rippled has in its database. This may be a disjoint sequence, for example `24900901-24900984,24901116-24901158`. |
 | `hostid`                            | String                    | On an admin request, returns the hostname of the server running the `rippled` instance; otherwise, returns a unique four letter word. |
 | `io_latency_ms`                     | Number                    | Amount of time spent waiting for I/O operations, in milliseconds. If this number is not very, very low, then the `rippled` server is probably having serious load issues. |
 | `last_close`                        | Object                    | Information about the last time the server closed a ledger, including the amount of time it took to reach a consensus and the number of trusted validators participating. |
 | `load`                              | Object                    | *Admin only* Detailed information about the current load state of the server |
 | `load.job_types`                    | Array                     | *Admin only* Information about the rate of different types of jobs the server is doing and how much time it spends on each. |
 | `load.threads`                      | Number                    | *Admin only* The number of threads in the server's main job pool. |
-| `load_factor`                       | Number                    | The load factor the server is currently enforcing, as a multiplier on the base transaction cost. For example, at `1000` load factor and a reference transaction cost of 10 drops of XRP, the load-scaled transaction cost is 10,000 drops (0.01 XRP). The load factor is determined by the highest of the individual server's load factor, cluster's load factor, and the overall network's load factor. **Note:** This `load_factor` is calculated as the ratio of the `load_factor` and the `load_base` that are reported by the [`server_state` command](#server-state) |
+| `load_factor`                       | Number                    | The load factor the server is currently enforcing, as a multiplier on the base transaction cost. For example, at `1000` load factor and a reference transaction cost of 10 drops of XRP, the load-scaled transaction cost is 10,000 drops (0.01 XRP). The load factor is determined by the highest of the [individual server's load factor](concept-transaction-cost.html#local-load-cost), the cluster's load factor, the [open ledger cost](concept-transaction-cost.html#open-ledger-cost) and the overall network's load factor. |
```

Comment:
On further thought, would it be worth including that this is the "load-scaled open ledger transaction cost" or something like that?  `load_factor_server` could then note that factor is sufficient to get into the transaction queue.  The link to "open ledger cost" is probably sufficient, but I'm not sure.